deployment/cloud-run/unified_ai_api.py

```
@@ -0,0 +1,2261 @@
+            "Complete journal processing pipeline",
+        ],
+    }
+
+
+if __name__ == "__main__":
+    port = int(os.environ.get("PORT", "8080"))
+    # Security: Only bind to all interfaces in production environments
+    # Default to localhost for development to avoid exposure
+    host = os.environ.get("HOST", "127.0.0.1")
+    if os.environ.get("PRODUCTION") == "true" or os.environ.get("CLOUD_RUN_SERVICE"):
+        host = "0.0.0.0"  # Cloud Run and production environments
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)